CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/CVKSwapChain.java

```diff
@@ -160,6 +162,8 @@ public int Initialise() {
             if (VkFailed(ret)) return ret;
```

Comment:
We need to rename these.  Doesn't need to happen in this change.

Reply:
Yep I agree, I need to do a pass on the CommandBuffer file as well. Will do in another change